Tools/build/.ruff.toml

```diff
@@ -24,10 +32,6 @@ ignore = [
     "UP038",   # Use `X | Y` in `isinstance` call instead of `(X, Y)`
 ]
 
-[per-file-target-version]
-"deepfreeze.py" = "py310"
-"stable_abi.py" = "py311"  # requires 'tomllib'
```

Comment:
Moved to the top of the file as currently this is in-between the `lint` table and a `lint` subtable